### DISCUSSION TITLE
Pruebas de Hacker Rank, letcode y otros

### DISCUSSION THREAD

**Discussion by gjordanra** [Launch-X-Latam/MisionFrontEnd#151] [Q&A]
Se que hay mucho aca 👌 que saben programar y andamos en esto de buscar trabajo, para ver si alguno me puede ayudar y explicar como testear los corner cases(las ocultas) en las pruebas de desarrollo de estas plataformas.
Se les agradece! ✔👀

**Comment by VictorFalconi** [comment]
Como asi? A que te refieres? Entre a esas paginas y pues cuando submiteas tu solucion, se evaluan esos casos y si pasas, tu solucion se publica y te dan puntos

**Reply by gjordanra** [reply]
Cuando haces un test laboral, no te pasan solo esos casos de prueba, te ponen test adicionales fuera de lo común, corner cases o algo así.

**Reply by VictorFalconi** [reply]
Ah, ya entendi. Te dieron un enlace de leetcode de una empresa para la entrevista? A eso te refieres? Pues ni idea, ahi si no tengo experiencia :'v
